code/game/gamemodes/cult/runes.dm

```diff
@@ -945,6 +947,10 @@ var/list/teleport_runes = list()
 	notify_ghosts("Manifest rune created in [get_area(src)].", 'sound/effects/ghost2.ogg', source = src)
 
 /obj/effect/rune/manifest/can_invoke(mob/living/user)
+	if(is_busy)
+		to_chat(user,"<span class='cultitalic'>The rune is already in use.</span>")
```

Comment:
Hmm, this makes it so you can only manifest a single ghost from a single rune, and you can only have one rune a tile - meaning you can only manifest one ghost at a time. This seems like a rather hefty balance change to the rune